.github/workflows/publish-rustdocs.yml

```diff
@@ -43,12 +49,51 @@ jobs:
             -v "${GITHUB_WORKSPACE}:/workspace" \
             -v "${HOME}/.cargo:/root/.cargo" \
             -w /workspace \
-            -e RUSTDOCFLAGS='-D warnings' \
```

Comment:
Isn't this env propagated inside the Docker container and inside the `bash` program?

Reply:
moved it up